org.jacoco.build/pom.xml

```diff
@@ -553,6 +558,30 @@
         </executions>
       </plugin>
 
+      <plugin>
+        <groupId>com.diffplug.spotless</groupId>
+        <artifactId>spotless-maven-plugin</artifactId>
+        <configuration>
+          <java>
+            <includes>
+              <include>src/**/*.java</include>
+            </includes>
```

Comment:
I would like to propose the follwing configuration:

    <includes>
      <include>**/*.java</include>
    </includes>
    <excludes>
      <exclude>target/**/*</exclude>
    </excludes>

This would basically check every `*.java` file in our tree, especially code in `jacoco-maven-plugin.test`which is currently not covered.

Reply:
@marchof I added `.gitattributes` and configured Spotless to check that Java files use `LF` as line endings.

Unfortunately
```
<include>**/*.java</include>
```
causes weird reformatting of files in `jacoco-maven-plugin.test` - for example
```
jacoco-maven-plugin.test/it/it-multi-module/child/src/main/java/Example.java
```
but not
```
jacoco-maven-plugin.test/it/it-multi-module/child/src/main/java/somepackage/Example.java
```
My best guess is that in absence of `package` declaration first comment is considered as javadoc rather than as header. Maybe we can move files and add `package` declaration or add some javadoc. Not sure what will better. So I propose to merge this as is and investigate formatting of `jacoco-maven-plugin.test/it` later separately.